pkg/user/admin/add_pending.go

```diff
@@ -0,0 +1,26 @@
+package admin
+
+import (
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common/flags"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/telemetry"
+	"github.com/urfave/cli/v2"
+)
+
+func AddPendingCmd() *cli.Command {
+	addPendingCmd := &cli.Command{
+		Name:      "add-pending-admin",
+		Usage:     "user admin add-pending-admin <AccountAddress> <AdminAddress>",
```

Comment:
`user admin add-pending-admin <AccountAddress> <AdminAddress>`

this means `<AccountAddress>` and `<AdminAddress>` are arguments not flags. But you have defined flags below
so this usage text should change to `user admin add-pending-admin --account-address <AccountAddress> --admin-address <AdminAddress>`